resources/js/components/Auth/AuthenticationCardLogo.vue

```diff
@@ -6,56 +6,10 @@
       style="backdrop-filter: blur(10px);"
     >
       <svg
```

Comment:
since this file is showing the same logo in `Logo.vue` I think it's best to include the logo component instead. please consider this one.